frontend/app/src/components/home/ChatMessage.svelte

```diff
@@ -119,6 +120,9 @@
     $: msgUrl = `/#/${chatId}/${msg.messageIndex}?open=true`;
     $: isProposal = msg.content.kind === "proposal_content";
     $: inert = deleted || collapsed;
+    $: undeletingMessagesStore = client.undeletingMessagesStore;
+    $: undeleting = $undeletingMessagesStore.has(msg.messageId) ?? false;
```

Comment:
Surely `$undeletingMessageStore.has(msg.messageId)` will _always_ return a boolean and therefore the `??` is redundant.

Reply:
If I don't do that then the compiler thinks undeleting is `boolean | undefined`

Reply:
Hmmm - not when _I_ do it. `$: undeleting = $undeletingMessagesStore.has(msg.messageId);` seems to work fine.